### PR TITLE
stuff added

### PR DESCRIPTION
**Pull request by Rumthorp**
test stuff

### CODE REVIEW

File: part-2-server/server.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+var http = require("http");
+var fs = require("fs");
+
+var server = http.createServer(function(req, res) {
```
Comment (nmadd) [inline part-2-server/server.js:4]: Creative use of regex

Comment (nmadd) [inline part-2-server/server.js:4]: Why is this a separate if statement and not part of the switch statement?

File: part-2-server/server.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        var target = req.url.split("");
+        target.shift();
+        target = target.join("");
+        target = target + ".txt"
+        fs.readFile(target, function(err, data) {
+            res.write(data.toString());
+            res.end();
+        })
+    }
+
```
Comment (nmadd) [inline part-2-server/server.js:15]: This is a little hard to follow...would be helpful to either save more descriptive variables, or separate this logic out as a separate function

Comment (nmadd) [reply part-2-server/server.js:15]: URL requests that start with 'poem' will hit the first if statement, then will hit the switch statement and eventually hit the default case. This will cause your server to crash because within the first if statement it will try to read the file asyncronously, continue on to the switch statement, then set a response.end before the original file is read

Comment (nmadd) [reply part-2-server/server.js:15]: Keep it simple...if you decide to use a switch statement, keep everything within the switch statement. If that's not possible, avoid using a switch statement and use an if/else instead

File: part-1-ajax/test.js [hunk 3]
```diff
@@ -1,76 +1,90 @@
...
-
-  var button = document.getElementsByTagName('button')[0]
-  button.addEventListener('click', function(){
+// function parseData(country) {
+//     takes the country data
+//     extracts the name, capital, population, and neighboring countries
+//     iterates through the neighbors array to send off second AJAX request
+// }
 
+var button = document.getElementsByTagName('button')[0]
```
Comment (ckuang) [inline part-1-ajax/test.js:14]: good job. but in the future try to slim down your code. for example, the success callbacks for each ajax request could be defined elsewhere for better readability.
